frame/staking/src/pallet/impls.rs

```diff
@@ -719,7 +720,6 @@ impl<T: Config> Pallet<T> {
 
```

Comment:
The comments at https://github.com/paritytech/substrate/blob/32abede3de9d86461a045e2f247f0c45edc2826a/frame/staking/src/pallet/impls.rs#L709 are now outdated

Reply:
thank you.